webtau-docs/webtau/reference/table-data.md

```diff
@@ -0,0 +1,23 @@
+# Create
+
+Use language specific DSL to create `TableData` instance:
+
+```tabs
+Groovy:
+:include-groovy: com/twosigma/webtau/data/table/TableDataExtensionTest.groovy {entry: "createTableWithUnderscore", bodyOnly: true}
+
+Java:
+:include-groovy: com/twosigma/webtau/data/table/TableDataTest.groovy {entry: "createTableDataInOneGo", bodyOnly: true}
+
+ Note: example above assumes `import static com.twosigma.webtau.Ddjt.*`. Additionally `Ddjt` has `___` defined of various lengths. 
```

Comment:
The example above assumes ...

Additionally, Ddjt has header-separating lines defined using underscores `____` of various lengths, which you can optionally use for aesthetics.